foss4gna_24/.ipynb_checkpoints/big-EO-viz-checkpoint.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
<p>Spell out BCI, maybe mention it's relevance to Smithsonian and forest research?</p><p>I assume we have permissions to show public the AGB estimate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/geotrees-demo/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Updated with-</p><p> </p><blockquote>The BCI is an important Smithsonian Tropical Research Institute (STRI) research site for studying the tropical forests.</blockquote><blockquote><br></blockquote><blockquote>Reference:</blockquote><blockquote>[Barro Colorado Island](<a href="https://stri.si.edu/facility/barro-colorado)" target="_blank">https://stri.si.edu/facility/barro-colorado)</a></blockquote>